litellm/types/guardrails.py

```diff
@@ -105,6 +106,9 @@ class LitellmParams(TypedDict):
     guard_name: Optional[str]
     default_on: Optional[bool]
 
+    vendor_specific_params: Optional[Dict]
```

Comment:
looking at everything that is in here, we should go for either one of two things:

- `vendor_params` (the other one is too long and descriptive) if we would like to suggest to keep it generic and for other ones to reuse it
- `acuvity_params` if we want to denote that this is truly just for us. by the looks of it, every other vendor has also just placed their things shamelessly into this class after all

Reply:
lol... :D ... yes, every vendor is just slapping their variables in the class.